README.md

```diff
@@ -39,19 +39,14 @@ With the ESnext field declarations proposal, the above example can be written as
 class Counter extends HTMLElement {
```

Comment:
Those aren't equivalent, because the former has 1 prototype method that's shadowed by a bound function on N instances; and the latter has N instance arrow functions.

A proper @bound decorator (for React, at least) would go on a prototype method and not use class fields at all.